grovel/grovel.lisp

```diff
@@ -802,8 +802,12 @@ string."
     (process-wrapper-form out form)))
 
 (define-wrapper-syntax in-package (name)
-  (setq *package* (find-package name))
-  (push `(in-package ,name) *lisp-forms*))
+  (let ((desired-package (find-package name)))
+    (assert desired-package ()
+            "Wrapper file specified (in-package ~s)~%however that does not name a known package"
```

Comment:
@luismbo I like it having 'Wrapper' at the beginning as it sets the context for the rest of the message quickly. I generally want to mentally narrow down an issue as fast as possible